docker-compose.yaml

```diff
@@ -6,31 +6,6 @@ services:
     image: ghcr.io/opensafely/job-runner:latest
     init: true
     restart: unless-stopped
-    environment:
-      &jobrunner_environment
-      # The job server endpoint
-      - JOB_SERVER_ENDPOINT=${OPENSAFELY_JOB_SERVER_ENDPOINT}
-      # The backend requests which we should monitor
-      - BACKEND=${OPENSAFELY_BACKEND}
-      # Credentials for acessing the job server
-      - JOB_SERVER_TOKEN=${OPENSAFELY_JOB_SERVER_TOKEN}
-      # Working directory (see the volume mounts below)
-      - WORK_DIR=/work_dir
```

Comment:
This still needs setting as it's not in the `.env` file. That's because it's a reference to a mount point defined within docker-compose that doesn't make sense in any other context.